.github/workflows/release.yml

```diff
@@ -0,0 +1,69 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Build Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - uses: olegtarasov/get-tag@v2
+        id: tagName
+
+      - name: Checkout build tools
+        uses: actions/checkout@v2
+        with:
+          repository: EllisLab/ExpressionEngine-Build
```

Comment:
Having it listed here just exposes its name, it does not provide any kind of access to external people or machines

Reply:
Okay - I wasn't sure if this would run into problems accessing that repo since it is a private repo in a different organization. From outside conversation, we determined that this does appear to work, and if any permissions would need to be adjusted, that would be on the github administration side fo things.